### PR TITLE
Local docker compose

### PR DESCRIPTION
**Pull request by haiodo**
Allow to use docker compose configuration to have a working platform instance locally. And also allow to use instances setup for local development.

Signed-off-by: Andrey Sobolev [haiodo@gmail.com](mailto:haiodo@gmail.com)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,8 +12,52 @@ You need Microsoft's [rush](https://rushjs.io) to install application. Install [
...
+After all will be setup and running, we need to create workspace/account and associate account to workspace.
+
+```bash
+cd ./dev/tools
+rushx run-local create-workspace ws1 -o DevWorkspace # Create workspace
+rushx run-local create-account user1 -p 1234 -f John -l Appleseed # Create account
+rushx run-local assign-workspace user1 ws1 # Assign worksapce to user
+```
+
+Now it is possible to check how it is running by going into 'http://localhost:8081/login%3Acomponent%3ALoginApp' and logic with user credentials above.
```
Comment (BykhovDenis) [inline README.md]: Typo

**Review by BykhovDenis — APPROVED** [review]

File: dev/tool/package.json [hunk 2]
```diff
@@ -9,9 +9,10 @@
     "build:watch": "tsc",
     "lint:fix": "eslint --fix src",
     "start": "ts-node src/index.ts",
-    "bundle": "esbuild src/index.ts --bundle --minify --platform=node > bundle.js",
+    "bundle": "esbuild src/index.ts --bundle --minify --platform=node > bundle.js",
```
Comment (sumboid) [inline dev/tool/package.json:12]: trailing spaces

**Review by sumboid — APPROVED** [review]

File: server/front-ui/src/platform.ts [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+async function loadAccountUrl (): Promise<string> {
+  return await new Promise<string>((resolve) => {
+    const xmlHttp = new XMLHttpRequest()
+    xmlHttp.onreadystatechange = function () {
+      if (xmlHttp.readyState === 4 && xmlHttp.status === 200) {
+        resolve(xmlHttp.responseText)
+      }
+    }
+    xmlHttp.open('GET', '/account-url', true)
+    xmlHttp.send(null)
```
Comment (aplatoff) [inline server/front-ui/src/platform.ts]: Что за 2005? :) почему `XMLHttpRequest` а не `fetch`? :)

Comment (haiodo) [reply server/front-ui/src/platform.ts]: Fixed.

**Pull request merged**